src/applications/disability-benefits/526EZ/config/form.js

```diff
@@ -146,7 +146,6 @@ const formConfig = {
       pages: {
         veteranInformation: {
           title: 'Veteran Information', // TODO: Figure out if this is even necessary
-          description: 'This is the personal information we have on file for you.',
```

Comment:
This change and the next change didn't technically need to be in this PR, but they were bugging me. Page-level `description` properties aren't a thing.